scripts/env/rpmbuild/init.sh

```diff
@@ -57,6 +57,9 @@ install_cortx_py_utils() {
   pip3 install -r "$PWD/opt/seagate/cortx/utils/conf/python_requirements.txt" --ignore-installed
   pip3 install -r "$PWD/opt/seagate/cortx/utils/conf/python_requirements.ext.txt" --ignore-installed
 
+  # uninstall the numpy-1.19.5 as py-utils required numpy==1.19.2
+  pip3 uninstall numpy==1.19.5 -y
```

Comment:
1)Do not specify any version 
2)Do it before we scan for py-utils
3)Uninstall everything using the requirement file?